inscriptions/ISic000484.xml

```diff
@@ -186,20 +191,36 @@
       <desc>Add Description Here</desc>
     </graphic>
   </surface>
-
     -->
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AR">
+                <ab>
+                    <lb n="1"/>clementissimo <g ref="#interpunct">·</g> victoriossimo
+                    <lb n="2"/><expan><abbr>d</abbr><ex>omino</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>n</abbr><ex>ostro</ex></expan> <g ref="#interpunct">·</g> <name>Flavio</name> <g ref="#interpunct">·</g> <name>Valerio</name> 
+                    <lb n="3"/><name>Constantino</name> <g ref="#interpunct">·</g> maximo
+                    <lb n="4"/>pio <g ref="#interpunct">·</g> felici <g ref="#interpunct">·</g> invicto <g ref="#interpunct">·</g> <expan><abbr>aug</abbr><ex>usto</ex></expan>
+                    <lb n="5"/><name>Betitius</name> <g ref="#interpunct">·</g> <name>Perpetuus</name>
+                    <lb n="6"/><expan><abbr>v</abbr><ex>ir</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>c</abbr><ex>larissimus</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>corr</abbr><ex>ector</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>prov</abbr><ex>inciae</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>Sicil</abbr><ex>iae</ex></expan>
+                    <lb n="7"/>devotus <g ref="#interpunct">·</g> numini maies
+                    <lb n="8" break="no"/>tatique <g ref="#interpunct">·</g> eius 
+                    <lb n="9"/>semper <g ref="#interpunct">·</g> dicatus
+                </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app>
+                        <note>text of CIL</note>
+                    </app>
+                </listApp>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="it" resp="#AR">
+                <p>Al più moderato e vittorioso, nostro signore Flavio Valerio Costantino, massimo, pio, fortunato, indomito, augusto, Betizio Perpetuo, vir clarissimus, riparatore della provincia di Sicilia, sempre devoto al suo Genio e alla sua autorità, ha dedicato (questo)</p>
             </div>
+            <div type="translation" xml:lang="en" resp="#AR">
+                <p>To the most merciful and victorious, our lord Flavius Valerius Constantinus, maximus, pious, fortunate, unconquered, augustus, Betitius Perpetuus, vir clarissimus, reformer of the Sicilian province, always faithful to his Genius and authority, dedicated (this)</p>
```

Comment:
'maximus' either rmains capitalised as part of the name, or else if an epithet should be translated like the others, 'greatest'
'corrector', in this period, in Italy/Sicily, signifies 'governor'. I would translate 'governor (corrector)'